shared/src/packages/drivers/libs/validation-schemas/driver-update-request-body.validation-schema.ts

```diff
@@ -15,10 +20,21 @@ const driverLicenseNumber = joi
     'string.empty': DriverValidationMessage.DRIVER_LICENSE_NUMBER_REQUIRED,
   });
 
-const driverUpdateRequestBody = joi.object<DriverUpdateRequestDto, true>({
+const driverUpdateRequestBody = joi.object<
+  DriverUpdateRequestDto<MultipartParsedFile | FileObject>,
+  true
+>({
   ...commonSignUpRules,
   driverLicenseNumber,
   truckIds: joi.array().items(joi.number()),
+  files: joi
+    .array()
+    .items(joi.object())
+    .min(1)
+    .required()
+    .messages({
+      'array.min': `Choose at least 1 ${pluralizeString('file', 1)}`,
```

Comment:
and move this text to some validation rule enum
```suggestion
      'array.min': 'Choose at least 1 file',
```